### PR TITLE
link graphql stats service to k8 upstream

### PR DESCRIPTION
**Pull request by camallen**
link the graphql stats service to the kubernetes upstream

### CODE REVIEW

File: sites/swarm19a.conf [hunk 1]
```diff
@@ -75,8 +75,8 @@ server {
             rewrite ^/graphql / break;
```
Comment (adammcmaster) [inline sites/swarm19a.conf]: I think we'll need to remove this rewrite.

Comment (camallen) [reply sites/swarm19a.conf]: If we remove this we'll never get to the correct end point (GET / vs POST /graphql).

Comment (adammcmaster) [reply sites/swarm19a.conf]: I guess I don't understand why this is needed. What is the rewrite actually intended to do? If we leave it in, Kubernetes will receive `GET /` and route the request to the wrong service.

Comment (camallen) [reply sites/swarm19a.conf]: http://stats.zooniverse.org/graphql hits the root end point `/`
https://github.com/zooniverse/zoo-stats-api-graphql/blob/e8442c662b3112633cc3d80bdfd59b0cd94ad1b6/config/routes.rb#L6

Without this it'll try all GET requests will try to hit the non existant GET on `/graphql` https://github.com/zooniverse/zoo-stats-api-graphql/blob/e8442c662b3112633cc3d80bdfd59b0cd94ad1b6/config/routes.rb#L4

Comment (adammcmaster) [reply sites/swarm19a.conf]: Oh, so the graphql app doesn't respond to GET at all? That makes sense then.

**Comment by camallen** [comment]
Closing in favour of https://github.com/zooniverse/operations/pull/218 and just changing the host name

**Pull request closed without merging**